### PR TITLE
added CrowdSec Prometheus metrics URL

### PR DESCRIPTION
**Pull request by luisj1983**
Added CrowdSec metrics URL to prometheus.conf.

### CODE REVIEW

File: config/go.d/prometheus.conf [hunk 1]
```diff
@@ -1546,3 +1546,5 @@ jobs:
   - name: besu_local
     url: '127.0.0.1:9545'
     expected_prefix: 'besu_'
+  - name: crowdsec_local
+    url: http://127.0.0.1:6060/metrics
```
Comment (ilyam8) [inline config/go.d/prometheus.conf:1550]: Can you please add `expected_prefix`?

Comment (luisj1983) [reply config/go.d/prometheus.conf:1550]: Is there a syntax for multiple prefixes?

Comment (ilyam8) [reply config/go.d/prometheus.conf:1550]: @luisj1983 we need just one that would be unique for crowdsec. It is ok if this exporter has metrics with other prefixes.

Comment (luisj1983) [reply config/go.d/prometheus.conf:1550]: @ilyam8 OK, done. Anything else I need to do prior to a merge?

**Review by ilyam8 — APPROVED** [review]

File: config/go.d/prometheus.conf (hunk 1 above)
Comment (ilyam8) [reply config/go.d/prometheus.conf:1550]: No, thanks

**Comment by ilyam8** [comment]
Thanks, @luisj1983.

**Pull request merged**